pkg/cri/sbserver/image_remove.go

```diff
@@ -33,6 +35,18 @@ import (
 // Remove the whole image no matter the it's image id or reference. This is the
 // semantic defined in CRI now.
 func (c *criService) RemoveImage(ctx context.Context, r *runtime.RemoveImageRequest) (*runtime.RemoveImageResponse, error) {
+	ctx, span := tracing.StartSpan(ctx, makeSpanName("RemoveImage"))
```

Comment:
Can we do this again in the instrumented service rather than duplicating functions here?